content/backstage/plugins/code-coverage.md

```diff
@@ -0,0 +1,61 @@
+---
+humanName: Code Coverage
+heading: 'Backstage Code Coverage Plugin'
+lead: 'See code coverage summaries for your software in Backstage'
+attribution:
+  text: Spotify
```

Comment:
Not Spotify.
This plugin was developed by

```
Co-authored-by: nissayeva [natashaaay@gmail.com](mailto:natashaaay@gmail.com)
Signed-off-by: alde [r.dybeck@gmail.com](mailto:r.dybeck@gmail.com)

```

Reply:
https://github.com/backstage/backstage/pull/5350

Reply:
Done